### ISSUE TITLE
Translations?

### ISSUE THREAD

**Issue by ninjaaron** [ninjaaron/replacing-bash-scripting-with-python#3]
I'm very pleased by the positive reception of this tutorial, and I've been especially surprised by the the amount of attention it's gotten from Chinese programmers. I was just thinking today that it would be nice if anyone wanted to do a translation into Chinese when I noticed, lo and behold @de8ug seems to be working on a fork with a Chinese translation!

I just wanted to put the word out to him/her and anyone else that I would be happy to help with the translation process in any way I can. English is my native language and I don't feel qualified to do any translations myself, but I'd be happy to support the efforts of anyone who does wish to translate the tutorial.

**Comment by ivanhercaz** [comment]
I could try to help with Spanish. I haven't many time, but I might contribute bit by bit.

Well I start to translate I will fork it and create a branch for the translation to Spanish.

Thank you for this awesome guide @ninjaaron!

**Comment by ninjaaron** [comment]
Sounds great, @ivanhercaz! No pressure, of course, but if you could make the translation as a separate document so I can pull from you, that would be awesome!

**Comment by nycsv** [comment]
Hi @ninjaaron! Recently I found your well-organized information. I really appreciate it! Can I translate this information into Korean as well?

**Comment by ninjaaron** [comment]
@eesungkim Yes! please do! Let me know if you need any support. Finding anything that doesn't make sense will also help improve the English version.

**Comment by ivanhercaz** [comment]
@ninjaaron, I am going to fork and work in the Spanish translation bit by bit. What structure would you like to follow for the translations?

1. `/translations/lang.rst`
2. `/translations/README.lang.rst`
3. `/langs/lang.rst`
4. `/README.lang.rst`

Of course if you have or prefer another one, tell me to follow in my fork!

**Comment by ninjaaron** [comment]
@ivanhercaz sorry for taking so long to get back on this! I kind of like number 3. Short and to the point.

**Comment by ninjaaron** [comment]
Oh! I just looked at your fork and I see you're already using convention no. 1. That's fine, too. No need to change it.

**Comment by ivanhercaz** [comment]
> Oh! I just looked at your fork and I see you're already using convention no. 1. That's fine, too. No need to change it.

Oh! Don't worry @ninjaaron. I just use option 1 because it was the first in which I though before to comment in this issue the possible names. In 79e2830 I changed the name as you suggested.
